src/App.vue

```diff
@@ -577,22 +536,41 @@ export default {
             role.roleType === 'Director').length !== 0)
           this.storeDirectors({ data: { directors: [...directors] } })
 
-          this.setFilings([{ filing }])
+          // build display name
+          const type: string = this.getCorpTypeDescription(this.entityType)
+          const name: string = this.filingTypeToName(FilingTypes.INCORPORATION_APPLICATION)
+          const desc: string = this.entityName || this.getCorpTypeNumberedDescription(this.entityType)
+
+          const effectiveDate: Date = this.apiToDate(filing.header.effectiveDate)
+
+          // add this as a filing (for Filing History List)
+          const filingItem: LedgerIF = {
```

Comment:
I created [# 8692](https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8692) to refactor this.